source/Planet.cpp

```diff
@@ -511,7 +511,7 @@ const Wormhole *Planet::GetWormhole() const
 bool Planet::IsAccessible(const Ship *ship) const
 {
 	// If this is a wormhole that leads to an inaccessible system, no ship can land here.
-	if(wormhole && ship->GetSystem() && wormhole->WormholeDestination(*ship->GetSystem()).Inaccessible())
+	if(wormhole && ship && ship->GetSystem() && wormhole->WormholeDestination(*ship->GetSystem()).Inaccessible())
```

Comment:
further down is already a check if the ship exists. maybe you just want to move it up?

Reply:
I don't want that check above the check between this one and where it currently is.

Reply:
I guess you are right